src/defaults.mligo

```diff
@@ -4,15 +4,22 @@
 #include "types.mligo"
 #include "proposal.mligo"
 
-let validate_default_config (data : initial_config_data) : unit =
+let validate_proposal_flush_expired_time (data : initial_config_data) : unit =
   if data.proposal_expired_time <= data.proposal_flush_time then
-    failwith("proposal_expired_time needs to be bigger than proposal_flush_time")
+    failwith("'proposal_expired_time' needs to be bigger than 'proposal_flush_time'.")
   else if data.proposal_flush_time <= (data.period.length * 2n) then
-    failwith("proposal_flush_time needs to be more than twice the period length")
+    failwith("'proposal_flush_time' needs to be more than twice the 'period' length.")
   else unit
 
+let validate_quorum_threshold_bound (data : initial_config_data) : unit =
+  if data.max_quorum >= data.quorum_threshold && data.quorum_threshold >= data.min_quorum then
+    unit
+  else
+    failwith("'quorum_threshold' value need be between 'max_quorum' and 'min_quorum'.")
```

Comment:
Let's be as detailed as possible here, I think we should split the check in two and give two different messages for "greater or equal then `min_quorum`" and "less than or equal to `max_quorum`".